sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -386,6 +511,36 @@ constant property.
 
 |===
 
+The following template implements the overloaded operators for types
```

Comment:
I think the use of the word "template" is confusing here given that it seems you're using it for its English definition when the term "template" has a meaning in C++ and the set of functions you're describing are themselves function templates.

Perhaps just add this as a single row in the table (then it's clear that it refers to a member function) but put operatorOP in italics and in the text in the right side column say something to the effect of "operatorOP can be any of ..." and list all the ops.